src/test/rgw/bucket_notification/__init__.py

```diff
@@ -30,6 +31,20 @@ def setup():
 
     global default_cluster
     default_cluster = defaults.get("cluster")
+    
+    version = defaults.get("version")
+    if version == "v1":
+        if default_cluster != 'noname':
+            print('WARNING: topic and notification multisite syncing is not enabled (v1)')
+    elif version == "v2":
+        _, result = admin(['zonegroup', 'modify', '--enable-feature=notification_v2'], default_cluster)
```

Comment:
do we need to update the zone as well? i tested this and it works even without updating the zone, but will it impact for any other scenario ? or since this tests are applicable only with single zone so its not a problem

Reply:
we only need the cluster and zonegroup. this is a single zone test.
testing the multisite syncing will happen in the multisite suite (the test added in the previous commit)